contracts/strategies/IStrategy.sol

```diff
@@ -63,19 +64,18 @@ interface IStrategy {
     // then checking the balance of the contract, and paying the difference
     function skim(address token) external;
 
+    /// @return Input the values you would send to distribute(), get the amounts each recipient in the array would receive
+    function getPayouts(address[] memory _recipientIds, bytes memory _data, address _sender)
```

Comment:
should this be view ?

Reply:
yes!
good catch

we also should talk about the return value
i think it should be a struct with an address and an amount. since the receipientId can differ from the address were the funds go to